plugins/removeAttrs.js

```diff
@@ -101,10 +101,10 @@ export const fn = (root, params) => {
     element: {
       enter: (node) => {
         for (let pattern of attrs) {
-          // if no element separators (:), assume it's attribute name, and apply to all elements *regardless of value*
+          // if no element separators (:), assume its attribute name, and apply to all elements *regardless of value*
```

Comment:
```suggestion
          // if no element separators (:), assume it's attribute name, and apply to all elements *regardless of value*
```

This one should be `it's`, short for `it is`. Though the comment in general isn't grammatically correct so I can see how this would be confusing.

Same applies to the other instance in the file.